src/Neo.Extensions/Neo.Extensions.csproj

```diff
@@ -15,7 +15,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Neo" />
     <InternalsVisibleTo Include="Neo.IO" />
-    <InternalsVisibleTo Include="$(AssemblyName).Tests" />
+    <InternalsVisibleTo Include="Neo.UnitTests" />
```

Comment:
is not Neo.Extensions.Tests?
```suggestion
    <InternalsVisibleTo Include="$(AssemblyName).Tests" />
```

Reply:
I made `Neo.Extensions` all the methods `public`, no need to have `internal`.

Reply:
@shargon